libs/java/auth_core/src/main/java/com/yahoo/athenz/auth/impl/UserAuthority.java

```diff
@@ -36,6 +36,7 @@
     private static final Logger LOG = LoggerFactory.getLogger(UserAuthority.class);
     static final String ATHENZ_PROP_PAM_SERVICE_NAME = "athenz.auth.user.pam_service_name";
 
+    final String userDomain = System.getProperty("athenz.user_domain", "user");
```

Comment:
Not sure what is the reason for this? Is there is a reason why you want to configure the domain name? Most likely you need to define your own authority as oppose to changing the default user authority.